doc/breakages.md

```diff
@@ -22,3 +22,4 @@ This list is largely advisory and items may be reevaluated once the time comes.
 * remove `DIR_DELIM` from Lua
 * stop reading initial properties from bare entity def
 * change particle default blend mode to `clip`
+* remove built-in knockback from `PlayerSAO:punch()`
```

Comment:
Sorry, but this is still unclear to me. Am I correct in understanding this as "remove built-in knockback completely, but keep `core.calculate_knockback`"? (That would make sense to me.)

(The reason I'm a bit confused is that knockback is implemented via `register_on_punchplayer`. The proposal isn't to only get rid of knockback for programmatic `objref:punch(...)` calls on the Lua side, right? So basically: Was this intended as `PlayerSAO::punch` (C++) or `PlayerRef:punch` (`ObjectRef:punch`) (Lua)?)

Reply:
the only knockback we have is for players, so this would be akin to deleting `builtin/game/knockback.lua`.

Reply:
```suggestion
* remove built-in knockback and related functions entirely
```

Reply:
Yeah I know, I was just wondering about punches initiated by mods vs punches initiated by clients. The suggestion clarifies that. Seems good to me.